wrappers/models_configurations/dalle_config.py

```diff
@@ -0,0 +1,9 @@
+from .base_config import BaseConfiguration
+from .config_types import ConfigTypes
+
+class DALLEConfiguration(BaseConfiguration):
+    def __init__(self, api_key: str, images: int, resolution: str) -> None:
+        super().__init__(api_key)
+        self.config_type = ConfigTypes.DALLE 
+        self.images = images
```

Comment:
confusing name, should indicate it's number of images

Reply:
changed to `num_of_images`